### PR TITLE
CAL1-3 Facility Search Results Showing LIS and CWS Facility as the Source

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CAL1-3

#### Brief feature description
- this PR is a bug fix for this PR - https://github.com/ca-cwds/CALS/pull/292

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- no

### CODE REVIEW

**Comment by ratneshraval** [comment]
Please update PR name to include story number.

**Comment by Nasir2712** [comment]
@ratneshraval updated

File: app/javascript/search/common/commonUtils.js [hunk 1]
```diff
@@ -43,3 +43,7 @@ export const fullName = (object) => {
 export const checkValueForNull = (value) => {
   return value == null ? 'N/A' : value
 }
+
+export const checkValueForNullOrEmptyString = (value) => {
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: So why is this needed over `checkValueForNull` ? If data is empty string, it should show empty string instead of 'N/A', right?

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: ok, i'll update it.
